api/argo-templates/render.go

```diff
@@ -53,8 +54,8 @@ func GetWrokerRebuildWorkflow(hostname string, xName string) ([]byte, error) {
 	tmpl, _ = tmpl.Parse(string(argoWorkflow))
 	var tmpRes bytes.Buffer
 	err = tmpl.Execute(&tmpRes, map[string]interface{}{
-		"TargetNcn":  hostname,
-		"TargetNcns": []string{hostname}})
+		"TargetNcn":  hostnames[0],
```

Comment:
Is there any chance this could be empty? I don't see any length validation in the function to ensure there is at least a first element.

Reply:
this will need a major rework. There is another ticket [CASMPET-5466](https://jira-pro.its.hpecorp.net:8443/browse/CASMPET-5466) which should address this one. it is hard coded for now